CONTRIBUTING.md

```diff
@@ -60,3 +128,16 @@ The format can be described more formally as follows:
 ```
 
 The first line is the subject and should be no longer than 70 characters, the second line is always blank, and other lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various git tools.
+
+Note: if your pull request isn't getting enough attention, you can use the reach out on Slack to get help finding reviewers.
+
```

Comment:
I am not  a native speaker, so just a soft suggestion, is it good to reword this sentence to something like this, "you can reach out to Slack (link here...) to ask for more attention from reviewers."?

Reply:
thanks for the feedback @chendave , the slack link has been provided on line 43 above.
I guess "reach out on Slack" sounds correct :)